src/main/java/frc/robot/commands/auto/PathPlannerCommand.java

```diff
@@ -25,12 +25,15 @@ public class PathPlannerCommand extends SequentialCommandGroup {
   Alliance alliance;
   
   public PathPlannerCommand(ArrayList<PathPoint> waypoints, Drivetrain drive) {
+    this(waypoints, drive, true);
+  }
+
+  public PathPlannerCommand(ArrayList<PathPoint> waypoints, Drivetrain drive, boolean useAllianceColor) {
     this(new ArrayList<PathPlannerTrajectory>(Arrays.asList(PathPlanner.generatePath(
       new PathConstraints(AutoConstants.kMaxAutoSpeed, AutoConstants.kMaxAutoAccel),
       waypoints.get(0),
-      waypoints.get(1),
-      (PathPoint[]) Arrays.copyOfRange(waypoints.toArray(), 2, waypoints.size())
-    ))), 0, drive, true);
+      waypoints.get(1)
+    ))), 0, drive, false, useAllianceColor);
```

Comment:
is there a reson why reset pose is not true by defult like it was before?

Reply:
Well if you are constructing points from your current pose no reason to reset pose to your current pose, that does nothing. And if your pose happens to change might be resetting to the wrong pose. The only time you should really reset your pose is if you are sure you are at the location, which only happens in auto (the first path)